### PR TITLE
Add Fedmenu

### PR DESCRIPTION
**Pull request by pypingou**

### CODE REVIEW

File: kerneltest/templates/master.html [hunk 1]
```diff
@@ -80,5 +80,22 @@ <h4 class="hide">Main Site Links:</h4>
       </div>
     </div>
 
+    {% block jscripts %}
+    <script type="text/javascript"
+        src="{{ url_for('static', filename='jquery-1.11.2.min.js') }}"></script>
```
Comment (ralphbean) [inline kerneltest/templates/master.html]: For the jquery bit here.. you could just use the one hosted by fedmenu?

File: kerneltest/templates/master.html [hunk 2]
```diff
@@ -80,5 +80,22 @@ <h4 class="hide">Main Site Links:</h4>
...
+
+    {% if config['FEDMENU_URL'] %}
+    <script src="{{ config['FEDMENU_URL'] }}/js/fedmenu.js"></script>
+    <script>
+      fedmenu({
+          'url': '{{ config["FEDMENU_DATA_URL"] }}',
+          'mimeType': 'application/javascript',
+          'position': 'bottom-right',
```
Comment (ralphbean) [inline kerneltest/templates/master.html]: And here, how about passing in ``'package': 'kernel',``?

File: kerneltest/templates/master.html (hunk 1 above)
Comment (pypingou) [inline kerneltest/templates/master.html]: Good point, I'll do that instead

**Comment by ralphbean** [comment]
:+1: from me, fwiw.

**Comment by pypingou** [comment]
@jmflinuxtx an opinion on this?

**Comment by jmflinuxtx** [comment]
Seems okay

**Pull request merged**